sdk/communication/azure-communication-phonenumbers/tests.yml

```diff
@@ -21,6 +21,9 @@ stages:
           MatrixReplace:
             - COMMUNICATION_SKIP_CAPABILITIES_LIVE_TEST=false/true
       Clouds: Public,Int
+      EnvVars:
+        AZURE_SKIP_LIVE_RECORDING: 'True'
+        AZURE_TEST_RUN_LIVE: 'true'
```

Comment:
Both should be "True" with capitalized T

Reply:
This is actually taken straight from the migration guide: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md#live-test-pipelines. I can make the change, but the thing is that I am not sure if this is used by Python or if the value is used by the pipeline script itself.

Reply:
You can align on either for consistency, but the values should actually be case-insensitive when used by pipelines